### ISSUE TITLE
about 30shots inference?

### ISSUE THREAD

**Issue by LXYrsy** [infinity7428/hANMCL#17]
When I use the provided model for 30 shots of the inference，support_im_paths = random.sample(support_im_paths, k=self.testing_shot) code error，can you tell me why?

**Comment by QuangHieu1704** [comment]
Seems like your support set doesn't have enough images for each class. Make sure you have at least 30 images each class in support set if you want to inference 30 shots.

**Comment by QuangHieu1704** [comment]
Yes, you can crop objects from other images in COCO dataset and add to your support set.

**Comment by LXYrsy** [comment]
but,the support set is DANA provided,if it can't work,How did they inference it?

**Comment by QuangHieu1704** [comment]
Why don't you ask them. But in my way, I add more images from COCO to support set myself and it works.

**Comment by LXYrsy** [comment]
Thank you. I'll try it now

**Comment by infinity7428** [comment]
I uploaded support images. It probably works just right for the numbers.
If not, please leave a comment again.
